Specs/DateTime/English/DateTimeModel.json

```diff
@@ -23962,5 +23962,304 @@
         }
       }
     ]
+  },
+  {
+    "Input": "schedule me a 1:1 meeting",
+    "Comment": "Taskmode supress 1:1 from getting categorised as time.",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript",
+    "Results": [
+      {
+        "Text": "1:1",
+        "Start": 14,
+        "End": 16,
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T01:01",
+              "type": "time",
+              "value": "01:01:00"
+            },
+            {
+              "timex": "T13:01",
+              "type": "time",
+              "value": "13:01:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "schedule me a 1:1 meeting on 14 nov",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript",
+    "Results": [
+      {
+        "Text": "1:1",
+        "Start": 14,
+        "End": 16,
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T01:01",
+              "type": "time",
+              "value": "01:01:00"
+            },
+            {
+              "timex": "T13:01",
+              "type": "time",
+              "value": "13:01:00"
+            }
+          ]
+        }
+      },
+      {
+        "Text": "14 nov",
+        "Start": 29,
+        "End": 34,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-11-14",
+              "type": "date",
+              "value": "2015-11-14"
+            },
+            {
+              "timex": "XXXX-11-14",
+              "type": "date",
+              "value": "2016-11-14"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "schedule me a 1:1 on saturday",
```

Comment:
Same and so on.

Reply:
it is extracted as time, maybe we could modify MinuteRegex in Base-DateTime from '[0-5]?\d' to '[0-5]\d' ? So the problem will be solved in all languages (the tests results are not affected).

Reply:
@aitelint that seems reasonable if the minute regex is not used in some other case that'd break. Can you PR the change?